Z/ZPares/build_tarballs.jl

```diff
@@ -0,0 +1,103 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "ZPares"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/cometscome/zpares_mirror.git", "ab453f5c3aa295bb43f6396e796db8d03964b5b3")
```

Comment:
Thank you for checking this pull request. There are two reasons not to use the original source. 
1. The download link of the original one is 
https://zpares.cs.tsukuba.ac.jp/?download=242
But, maybe due to this "?download=242", the builds always failed. I think this link comes from Wiki system of the original web site. 
2. I confirmed that the original code can not be called by ccall because the code use the structure with pointer variables written in the modern Fortran. So I wrote new wrapper code. This wrapper code is only available in my repo.

Reply:
> The download link of the original one is
> https://zpares.cs.tsukuba.ac.jp/?download=242
> But, maybe due to this "?download=242", the builds always failed. I think this link comes from Wiki system of the original web site.

You can always use a [`FileSource`](https://docs.binarybuilder.org/stable/reference/#BinaryBuilderBase.FileSource) with the `filename` keyword argument:
https://github.com/JuliaPackaging/Yggdrasil/blob/09d1ef9763e2ae7b2498b19ed2f70aff5ef4c5b7/S/SQLite/build_tarballs.jl#L11-L12

> I confirmed that the original code can not be called by ccall because the code use the structure with pointer variables written in the modern Fortran. So I wrote new wrapper code. This wrapper code is only available in my repo.

That'd be an additional source